.github/workflows/push-image.yml

```diff
@@ -60,82 +113,113 @@ jobs:
 
   push:
     name: Push
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
I see now that it was inconsistent before. One being latest and the other be 22.04. In that context making the two at least match does make more sent to me. At this point not sure if latest or 24.04 makes the most sense based on what is done in other repos?

Reply:
in other repos we use 22